db/migration/1687770112891-CreateOriginsTable.ts

```diff
@@ -0,0 +1,44 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
+
+export class CreateOriginsTable1687770112891 implements MigrationInterface {
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`-- sql
+        CREATE TABLE origins (
+            id INT AUTO_INCREMENT PRIMARY KEY,
+            datasetTitleOwid VARCHAR(512),
+            datasetDescriptionOwid TEXT,
+            datasetDescriptionProducer TEXT,
+            producer VARCHAR(255),
+            citationProducer TEXT,
+            datasetUrlMain TEXT,
+            datasetUrlDownload TEXT,
+            dateAccessed DATE,
+            datePublished DATE
```

Comment:
I think this should maybe be a text field since it can be just a year - what do you think?

Reply:
Varchar actually